homework/index.js

```diff
@@ -30,13 +33,94 @@
     return elem;
   }
 
+  // deletes the previous repository table if selected repository changes
+  function removeChildElements(parent) {
+    while (parent.firstChild) {
+      parent.removeChild(parent.firstChild);
+    }
+  }
+  // to show the information of selected repository in a table
+  function showRepoInfo(index, repoDiv, repoArray, containerDiv) {
+    removeChildElements(repoDiv);
```

Comment:
You are removing the repoDiv but not the contributorsDiv? It will add to the contributorsDiv on every change and not remove anything.

Reply:
Last week i could not solve this problem, but it is done now.